node/cmd/guardiand/node.go

```diff
@@ -641,6 +642,12 @@ func runNode(cmd *cobra.Command, args []string) {
 		if *suiWS == "" {
 			logger.Fatal("If --suiRPC is specified, then --suiWS must be specified")
 		}
+		// guardiand v2.16.0 shipped hardcoding "ws://" for the websocket url.
```

Comment:
I swear I commented on this yesterday, but I don't see it anywhere.

Anyway, I think this belongs in the Sui code, not here.

Reply:
What's funny, is that the original version was inside the sui watcher and then I decided to move it. I'll put it back and update this.

Reply:
@bruce-riley fixed in b543ea7fada6389e26ec63da90d56d934ff7e7e1